script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -66,7 +66,9 @@ The one who smashed the third floor window and came tumbling inside is―――
 [sha:0f11569f373404bf4509036059950659fb50ba2e]{
 -- Page 80, Offset 30766.
 -- 「――――――<Ⅹ|デケム>」
-"――――――Decem."
+// do we want to bring back the X or chi or whatever here?
+// is it intended as a christogram?
+"――――――%{i}Decem.%{/i}"
```

Comment:
I thought it was gematria? Petrikow call on whether it's important

Reply:
It's numerology stuff indeed.
I guess you could change it all caps for that ROMANVS feeling but other than that I don't see any reason to change it.